targets/PythonSdk/source/source/PlayFabSettings.py

```diff
@@ -0,0 +1,45 @@
+import PlayFabErrors
+import sys
+import traceback
+
+ProductionEnvironmentURL = "https://{titleId}.playfabapi.com{methodUrl}"
+TitleId = "" # You must set this value for PlayFabSdk to work properly (Found in the Game
+             # Manager for your title, at the PlayFab Website)
+DeveloperSecretKey = None # You must set this value for PlayFabSdk to work properly (Found in the Game
+                          # Manager for your title, at the PlayFab Website)
+
+# Client specifics
+AdvertisingIdType = "" # Set this to the appropriate AD_TYPE_X constant below
+AdvertisingIdValue = None # Set this to corresponding device value
+
+# DisableAdvertising is provided for completeness, but changing it is not
+# suggested
+# Disabling this may prevent your advertising-related PlayFab marketplace
+# partners from working correctly
+DisableAdvertising = False
+AD_TYPE_IDFA = "Idfa"
+AD_TYPE_ANDROID_ID = "Adid"
+
+class InternalSettings:
+    pass
+
+_internalSettings = InternalSettings()
+
+_internalSettings.EntityToken = None
+_internalSettings.ClientSessionTicket = None
```

Comment:
Add a comment that says this is auto-populated by any PlayFabClientApi.Login method.